From b02995d0900af96c54acbb1e884b25f9399e9afb Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 21 Jan 2022 15:12:25 +0000
Subject: [PATCH]sync_invert config in
 vc4-kms-dpi-generic

Both the base node and override set these parameters to 0,
so it made no difference. The base node should have been 1.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-dpi-generic-overlay.dts
@@ -1,5 +1,5 @@
 /*
- * vc4-kms-dpi-at056tn53v1-overlay.dts
+ * vc4-kms-dpi-generic-overlay.dts
  */
 
 /dts-v1/;
@@ -27,12 +27,12 @@
 					hfront-porch = <24>;
 					hsync-len = <72>;
 					hback-porch = <96>;
-					hsync-active = <0>;
+					hsync-active = <1>;
 					vactive = <480>;
 					vfront-porch = <3>;
 					vsync-len = <10>;
 					vback-porch = <7>;
-					vsync-active = <0>;
+					vsync-active = <1>;
 
 					de-active = <1>;
 					pixelclk-active = <1>;
